lib/ansible/module_utils/docker/swarm.py

```diff
@@ -259,7 +259,12 @@ def get_service_inspect(self, service_id, skip_missing=False):
             Single service information structure
         """
         try:
-            service_info = self.inspect_service(service=service_id)
+            service_info = self.inspect_service(service_id)
+        except NotFound as exc:
```

Comment:
Shouldn't the `if exc.status_code == 404: ...` be removed in the expect-block below if we are now using `NotFound`? 

https://github.com/docker/docker-py/blob/master/docker/errors.py#L13

Reply:
I've just removed it.